main.go

```diff
@@ -113,6 +121,9 @@ func locatePromoVersion(mcVersion string) (error, string) {
 // the path to the downloaded file
 func download(mcVersion, promoVersion string) (error, string) {
 	combinedVer := fmt.Sprintf("%s-%s", mcVersion, promoVersion)
+	if mcVersion == "1.7.10" {
+		combinedVer = fmt.Sprintf("%s-%s-%s", mcVersion, promoVersion, mcVersion)
+	}
```

Comment:
Oh yeah, I forgot they used to do that weird naming. Was it just 1.7.10 or all versions before that too?

Reply:
From what I've seen and experienced with it, only 1.7.10 got this weird naming.
Even the promotions file that this downloads has a bit of weirdness in it that made me remove 1 of the for loops when I added the "latest" flag. It kept using the wrong entry.